test/integration/tls/tls_app/backend/app.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
This app is pretty generic and the only difference is `route` name and backend endpoint. If we use a generic route name and provide the backend endpoint name via environment variable, we can use this app in other test cases too. I'd suggest making the app generic and moving it under `aws-app-mesh-controller-for-k8s/test/framework/apps`

Reply:
Slightly different in that there is a forced sleep in timeout app request handlers. Let me check if we can merge them with some modifications.